etl/resources/wikipedia_reader_resource.py

```diff
@@ -0,0 +1,39 @@
+import json
+from collections.abc import Iterable
+from pathlib import Path
+
+from dagster import ConfigurableResource
+from unidecode import unidecode
+
+from etl.models import wikipedia
+from etl.models.types import DataFileName
+
+
+class WikipediaReaderResource(ConfigurableResource):  # type: ignore
```

Comment:
Not really a resource, either.

It's fine if not all of your classes fit into one of Dagster's types (assets, resources, etc.). I have my assets delegate to other code like a `WikipediaReader`, because assets are kind of a pain to test.

Reply:
Yes, I have replaced this and similar resources with normal classes.